src/components/Homepage/Homepage.tsx

```diff
@@ -1,10 +1,12 @@
 import CheckIcon from "@material-ui/icons/Check"
 import { Link } from "react-router-dom"
+
 import "./style.css"
 
 export function Homepage() {
   return (
     <div className="homepage">
+      <section></section>
```

Comment:
Is this needed? 🤔

Reply:
I remember putting that in there for some reason, and now I've forgotten why. Deleted! Until I remember what I wanted to add that for...😬